Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -985,18 +987,18 @@ AXCoreObject::AccessibilityChildrenVector AccessibilityNodeObject::radioButtonGr
 
 unsigned AccessibilityNodeObject::headingTagLevel() const
 {
-    const auto& tag = tagName();
-    if (tag == h1Tag)
+    const auto& elementName = this->elementName();
```

Comment:
This probably shouldn't be `const auto&` anymore since the return type is uint16_t rather than an `const AtomString&` like tagName was.